src/Android/Xamarin.Android/Samples/GraphicsOverlay/AddGraphicsWithSymbols/AddGraphicsWithSymbols.cs

```diff
@@ -0,0 +1,171 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Android.App;
+using Android.OS;
+using Android.Widget;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using System.Drawing;
+
+namespace ArcGISRuntimeXamarin.Samples.AddGraphicsWithSymbols
+{
+    [Activity]
+    public class AddGraphicsWithSymbols : Activity
+    {
+        // Create and hold reference to the used MapView
+        private MapView _myMapView = new MapView();
+
+        // Create the graphics overlay
+        private readonly GraphicsOverlay _overlay = new GraphicsOverlay();
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+
+            Title = "Add graphics with symbols";
+
+            // Create the UI, setup the control references and execute initialization
+            CreateLayout();
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create the map
+            Map myMap = new Map(BasemapType.Oceans, 56.075844, -2.681572, 13);
+
+            // Add the map to the map view
+            _myMapView.Map = myMap;
+
+            // Add the graphics overlay to the map view
+            _myMapView.GraphicsOverlays.Add(_overlay);
+
+            // Call functions to create the graphics
+            CreatePoints();
+            CreatePolygon();
+            CreatePolyline();
+            CreateText();
+        }
+
+        private void CreatePoints()
+        {
+            // Create a red circle simple marker symbol
+            SimpleMarkerSymbol redCircleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, Color.Red, 10);
+
+            // Create graphics and add them to graphics overlay
+            var graphic = new Graphic(new MapPoint(-2.72, 56.065, SpatialReferences.Wgs84), redCircleSymbol);
```

Comment:
I would recommend using the ArcGIS Runtime 'Graphic' type rather than 'var':
Graphic graphic = new Graphic(new MapPoint(-2.72, 56.065, SpatialReferences.Wgs84), redCircleSymbol);